portfolio/src/main/webapp/js/comments.js

```diff
@@ -0,0 +1,137 @@
+/**
+ * Sends blog-title and user's prefered comment limit to the server
+ * to recieve all relevant comments
+ *
+ * Generates comment html using the recieved json and
+ * a delete button to remove each comment via id
+ */
+function getComments() {
+    const limitInput = document.getElementById('comment-limit');
+    const value = limitInput.options[limitInput.selectedIndex].value;
+  
+    let blogTitle = document.getElementById('blog-title').innerHTML;
+    blogTitle = blogTitle.split(' ').join('_'); // Clean the url
```

Comment:
Nit: can you provide an example of how the URL would be cleaned by this statement?  I.e., "Before: xxx.com --> After: xxx"